tests/integration/test_ethernet.py

```diff
@@ -61,6 +61,28 @@ def provider(request):
 def testnic1():
     veth_name = "testeth"
     try:
+        return_code = subprocess.call(
```

Comment:
you don't need to do a subprocess call for this - you can do
```python
if os.path.isfile("/etc/sysconfig/network-scripts/ifcfg-" + veth_name):
    os.unlink("/etc/sysconfig/network-scripts/ifcfg-" + veth_name)
```
unless you can't use `os` or `os.path` in this context?

Reply:
ok,right

Reply:
Updated. Thanks. It is wasteful to spawn another process indeed.